assets/js/feature-tours/ga4-reporting.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/*
+ * Internal dependencies
+ */
+import { VIEW_CONTEXT_MAIN_DASHBOARD } from '../googlesitekit/constants';
+import Link from '../components/Link';
+
+const ga4Reporting = {
+	slug: 'ga4Reporting',
+	contexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+	version: '1.98.0',
+	gaEventCategory: ( viewContext ) =>
+		`${ viewContext }_dashboard-ga4-reporting`,
+	steps: [
+		{
+			target: '.googlesitekit-data-block--conversions .googlesitekit-data-block__title',
+			title: __(
+				'See the new metrics from Google Analytics 4',
+				'google-site-kit'
+			),
+			content: createInterpolateElement(
+				__(
+					'"Conversions" have replaced "Goals", since "Goals" no longer exist in GA4. <a>Learn how to set up Conversions</a>',
+					'google-site-kit'
+				),
+				{
+					a: (
+						<Link
+							href="https://support.google.com/analytics/answer/12966437"
```

Comment:
Please use the `getGoogleSupportURL` for constructing all support URLs.

Reply:
Thanks. Added the new `SupportLink` component and updated a few other components to use the new support link component as well.